src/Compilers/CSharp/Test/Symbol/Symbols/Retargeting/NoPia.cs

```diff
@@ -1367,6 +1367,227 @@ public void GenericsClosedOverLocalTypes1_1()
+                                new CSharpCompilation[] { comp },
+                                null,
+                                new MetadataReference[] { },
+                                null);
+
+            Assert.Equal(SymbolKind.ErrorType, assemblies1[0].GlobalNamespace.GetMember<MethodSymbol>("C.Test1").ReturnType.Kind);
+
+            var assemblies2 = MetadataTestHelpers.GetSymbolsForReferences(
+                                new CSharpCompilation[] { },
+                                null,
+                                new MetadataReference[] { comp.ToMetadataReference() },
+                                null);
+
+            Assert.Equal(SymbolKind.ErrorType, assemblies2[0].GlobalNamespace.GetMember<MethodSymbol>("C.Test1").ReturnType.Kind);
+        }
+
+        [ClrOnlyFact]
+        public void CannotEmbedValueTuple()
+        {
+            string piaSource = @"
+using System;
+using System.Runtime.InteropServices;
+
+[assembly: Guid(""f9c2d51d-4f44-45f0-9eda-c9d599b58257"")]
+[assembly: ImportedFromTypeLib(""Pia1.dll"")]
+
+namespace System
+{
+    public struct ValueTuple<T1, T2>
+    {
+        public ValueTuple(T1 item1, T2 item2) { }
+    }
+}";
+
+            var pia = CreateCompilationWithMscorlib(piaSource, options: TestOptions.ReleaseDll, assemblyName: "pia");
+            pia.VerifyDiagnostics();
+
+            string source = @"
+public class C
+{
+    public System.ValueTuple<string, string> Test1()
+    {
+        throw new System.Exception();
+    }
+    public (int, int) Test2()
+    {
+        throw new System.Exception();
+    }
+}";
+
+            var comp1 = CreateCompilationWithMscorlib(source, options: TestOptions.ReleaseDll,
+                            references: new MetadataReference[] { new CSharpCompilationReference(pia).WithEmbedInteropTypes(true) });
+            comp1.VerifyDiagnostics(
+                // (8,12): error CS1768: Type 'ValueTuple<T1, T2>' cannot be embedded because it has a generic argument. Consider setting the 'Embed Interop Types' property to false.
+                //     public (int, int) Test2()
+                Diagnostic(ErrorCode.ERR_GenericsUsedInNoPIAType, "(int, int)").WithArguments("System.ValueTuple<T1, T2>").WithLocation(8, 12),
+                // (4,19): error CS1768: Type 'ValueTuple<T1, T2>' cannot be embedded because it has a generic argument. Consider setting the 'Embed Interop Types' property to false.
+                //     public System.ValueTuple<string, string> Test1()
+                Diagnostic(ErrorCode.ERR_GenericsUsedInNoPIAType, "ValueTuple<string, string>").WithArguments("System.ValueTuple<T1, T2>").WithLocation(4, 19)
+                );
+
+            var comp2 = CreateCompilationWithMscorlib(source, options: TestOptions.ReleaseDll,
+                            references: new MetadataReference[] { MetadataReference.CreateFromImage(pia.EmitToArray()).WithEmbedInteropTypes(true) });
```

Comment:
Sure. 
I also changed to use the `ToMetadataReference` helper on line 1492 for `CSharpCompilationReference(pia).WithEmbedInteropTypes(true)`.